pages/new-collective-onboarding-modal.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from 'react-apollo';
+import { Box, Flex } from '@rebass/grid';
+
+import OnboardingModal from '../components/onboarding-modal/OnboardingModal';
+import ErrorPage from '../components/ErrorPage';
+import SignInOrJoinFree from '../components/SignInOrJoinFree';
+import Page from '../components/Page';
+import { H1, P } from '../components/Text';
+import { withUser } from '../components/UserProvider';
+import Loading from '../components/Loading';
+import Container from '../components/Container';
+import { getCollectivePageQuery } from '../components/collective-page/graphql/queries';
+
+class NewCollectiveOnboardingPage extends React.Component {
+  static async getInitialProps({ query }) {
+    return {
+      slug: query && query.slug,
+      query,
+    };
+  }
+
+  static propTypes = {
+    query: PropTypes.object,
+    slug: PropTypes.string, // for addCollectiveCoverData
+    data: PropTypes.object, // from withData
+    loadingLoggedInUser: PropTypes.bool,
+    LoggedInUser: PropTypes.object,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      show: true,
+    };
+  }
+
+  setShow = bool => {
+    this.setState({ show: bool });
+  };
+
+  render() {
+    const { data, LoggedInUser, query } = this.props;
+    const { show } = this.state;
+    const collective = data && data.Collective;
+
+    if (!LoggedInUser) {
```

Comment:
**[:sparkles: **Enhancement**]** We should ideally use `AuthenticatedPage` for this, it's a superset of `Page` that will automatically render the loading grid and the SignIn form when required. See https://github.com/opencollective/opencollective-frontend/blob/c9caf09555c23a62d07878fe45099c46af7888e5/pages/markOrderAsPaid.js#L66for an example.

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Reply:
Will also keep this in mind for the next PR 🙌